src/ui/shared/UrlContext.ts

```diff
@@ -0,0 +1,82 @@
+export type AppMode = 'onboarding' | 'wallet';
+export type WindowType = 'popup' | 'tab' | 'dialog';
+export type WindowLayout = 'column' | 'page';
```

Comment:
Two of these aren't used outside

Reply:
Now all 3 are internal / not exported anymore ✅